### DISCUSSION TITLE
[2187220]逆转裁判456 王泥喜精选集（有D加密 暂时无效）

### DISCUSSION THREAD

**Discussion by Kukiisama** [pjy612/SteamManifestCache#423] [Share]
`appid:2187220`
```
2187221;d9865c4fe54425b3fd8b8bb95b5f3b1d87e4c6c02a3ff6777ae43f9c480ac365
```

清单在附件里
[逆转裁判456 王泥喜精选集.zip](https://github.com/pjy612/SteamManifestCache/files/14055346/456.zip)

一下没注意，这游戏貌似有D加密，咋删帖来着qwq

**Comment by pjy612** [comment]
D加密不影响收录，总有需要的时候。
解锁 和 下载 还有能不能玩，一直都是 三回事。
